doc/classes/BitMap.xml

```diff
@@ -64,6 +70,13 @@
 				[code]epsilon[/code] is passed to RDP to control how accurately the polygons cover the bitmap: a lower [code]epsilon[/code] corresponds to more points in the polygons.
 			</description>
 		</method>
+		<method name="resize">
+			<return type="void" />
+			<argument index="0" name="new_size" type="Vector2" />
+			<description>
+				Resizes the image to the given size of type [code]Vector2[/code].
```

Comment:
You don't need to mention argument type, as it already appears in the method's signature. The description could be something like this instead:
```suggestion
				Resizes the image to [code]new_size[/code].
```

Reply:
Okay, I changed it.